profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -113,66 +116,37 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
   if ($entity_type == 'node') {
     $node = $form_state['entity'];
     if (workbench_moderation_node_moderated($node)) {
+      // If the tmgmt_poetry module is disabled remove possibility to select
+      // multiple languages and by doing that follow the default tmgmt path.
+      if (!module_exists('tmgmt_poetry')) {
+        $form['languages']['#multiple'] = FALSE;
+      }
 
       $states = array_values(_tmgmt_workbench_available_states());
       $state = $node->workbench_moderation['current']->state;
       $translation_possible = in_array($state, $states, 'strict');
 
       $options = $form['languages']['#options'];
 
-      /** @var TMGMTJobItem[] $existing_jobs */
-      $existing_jobs = tmgmt_job_item_load_latest(
-        'workbench_moderation',
-        $entity_type,
-        $node->nid,
-        $node->translations->original
-      );
+      // Getting information about existing job items.
+      if (!isset($form_state['tmgmt_workbench']['existing_jobs'])) {
+        $form_state['tmgmt_workbench']['existing_jobs'] = _tmgmt_workbench_get_existing_jobs($node);
+      }
+      $existing_jobs = $form_state['tmgmt_workbench']['existing_jobs'];
 
       $always_show_direct_translation_links = variable_get('tmgmt_workbench_moderation_always_show_direct_translation_links', FALSE);
 
       // Setting up an array with language codes based on the available options.
       $lang_codes = array_keys($options);
-      $language_requested = FALSE;
-
-      // NEXTEUROPA-12429 - additional header with last change date.
-      $form['languages']['#header'][TMGMT_WORKBENCH_REQUEST_LAST_CHANGE_INDEX] = t('Last change');
-      foreach ($form['languages']['#options'] as $key => $src_lang) {
-        $form['languages']['#options'][$key][] = '';
-        break;
-      }
 
       foreach ($lang_codes as $lang_code) {
-        // NEXTEUROPA-12429 - additional column with last change date.
-        // By default the last change date column is empty.
```

Comment:
@imanoleguskiza I removed them to increase readability of the code before refactoring. Initial code has been moved (not only from one file to another but also across the functions) and I thought that leaving comments could be a bit misleading.